app/backend/src/application/task/create-task.ts

```diff
@@ -9,13 +10,34 @@ import TaskSchema from 'database/models/task';
  * @param [projectId] - task's project identifier
  * @param [parentId] - id of the parent task (for subtask)
  * @param [assignees] - an array of task assignees
+ * @param statusId - id of the status the task has
  */
-export async function createTask(text: string, orderScore: number, projectId?: string, parentId?: string, assignees?: string[]): Promise<Task> {
-  return await TaskSchema.create({
+export async function createTask(text: string, orderScore: number, projectId?: string, parentId?: string, assignees?: string[], statusId?: string): Promise<CreateTaskResponsePayload> {
```

Comment:
same thing with the `CreateTaskResponsePayload`. App level should not operate with transport payload.